contracts/CryptoChampions.sol

```diff
@@ -427,10 +453,28 @@ contract CryptoChampions is ICryptoChampions, AccessControl, ERC1155 {
     }
 
     /// @notice Fetches the data of a single elder spirit
-    /// @param elderId The id of the elder being searched for 
+    /// @param elderId The id of the elder being searched for
     /// @return The elder's attributes in the following order (valid, raceId, classId, affinity)
-    function getElderSpirit(uint256 elderId) isValidElderSpiritId(elderId) external override view returns (bool, uint256, uint256, string memory) {
+    function getElderSpirit(uint256 elderId)
+        external
+        view
+        override
+        isValidElderSpiritId(elderId)
+        returns (
+            bool,
+            uint256,
+            uint256,
+            string memory,
+            int256
+        )
+    {
         ElderSpirit memory elderSpirit = _elderSpirits[elderId];
-        return (elderSpirit.valid, elderSpirit.raceId, elderSpirit.classId, elderSpirit.affinity);
```

Comment:
Just curious, does Solidity have something like Prettier where files get formatted on save?

Reply:
Yes it does I have it configured on my system. I can share my vs code settings later if you want.

Reply:
@Oozyx yes please.